src/components/providers/NotificationProvider.tsx

```diff
@@ -8,8 +8,9 @@ import { NotificationContextType } from '../../types/contexts'
 const NotificationProvider = ({ children }: PropsWithChildren<any>) => {
   const [notification, setNotification] = useState<Notification | undefined>(undefined)
 
-  const triggerNotification = (message: string, type: NotificationType) => {
-    setNotification({ message, type, visible: true })
+  const triggerNotification = (message: string, type: NotificationType, autoHideDuration?: number | null) => {
+    autoHideDuration = autoHideDuration === undefined ? 3000 : autoHideDuration
+    setNotification({ message, type, visible: true, autoHideDuration })
   }
```

Comment:
I think it would be easier to use a Boolean for disabling `autoHide`. This would not break backwards compatibility (and I don't think we're going to use different durations).

Reply:
done in pr #94